vignettes/dashboards/index.Rmd

```diff
@@ -213,7 +213,7 @@ ui <- page_navbar(
 shinyApp(ui, server)
 ```
 
-Be wary that `page_navbar()`'s `sidebar` argument puts the same sidebar on every page, which may or may not be desirable. In the [sidebar article](../sidebars), you'll learn some strategies for keeping the sidebar's contents close to the stuff they effect. Specifically, that article covers how to: (1) [hide/show `sidebar` contents on particular pages](../sidebars#conditional-contents) (2) [put a particular `sidebar()` with a given page](../sidebars#multi-page-example), and even (3) [put sidebar layouts inside of a `card()`](../sidebars#multi-page-example).
+Be wary that `page_navbar()`'s `sidebar` argument puts the same sidebar on every page, which may or may not be desirable. In the [sidebar article](../sidebars), you'll learn some strategies for keeping the sidebar's contents close to the elements they affect. Specifically, that article covers how to: (1) [hide/show `sidebar` contents on particular pages](../sidebars/index.html#conditional-contents), (2) [put a particular `sidebar()` with a given page](../sidebars/index.html#multi-page-example), and even (3) [put sidebar layouts inside of a `card()`](../sidebars/index.html#multi-page-example).
```

Comment:
Were the links problematic?

Reply:
Yes, the links only go to the top of the page, they were not going to a specific section